### DISCUSSION TITLE
2019 Lay-Z-Spa St. Lucia model #54327

### DISCUSSION THREAD

**Discussion by skyluker** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#234] [Show and tell]
I just want to confirm that 6-wire module works like a charm. (2019?)

@visualapproach Thank you for a great piece of work!
![20220316_215101](https://user-images.githubusercontent.com/28607288/158690116-fca60858-2b4d-4f3c-b38a-8975418fdc46.jpg)

**Comment by visualapproach** [comment]
Thank you!
